packages/design-system/src/components/ChoiceList/__snapshots__/Choice.test.tsx.snap

```diff
@@ -13,7 +13,7 @@ exports[`Choice applies className to label 1`] = `
 exports[`Choice has a hint and requirementLabel 1`] = `
 <FormLabel
   component="label"
-  fieldId="checkbox_foo_10"
+  fieldId="checkbox_foo_11"
```

Comment:
These unit tests really need to be refactored so they don't have so many snapshots, but that's out of scope for a last-minute feature like this